### ISSUE TITLE
Cannot add new git+https repo with helm v3.0.0 and helm-git v0.4.2

### ISSUE THREAD

**Issue by mojochao** [helm/helm#7025]
With helm v2.16.1 I am able to run the following:

```
$ helm repo add coresvcs-charts 'git+https://bitbucket.org/sambatv/coresvcs-charts@?ref=master'
```

With helm v3.0.0 I get an error:

```
$ helm repo add coresvcs-charts 'git+https://bitbucket.org/sambatv/coresvcs-charts@?ref=master'
Error: unknown flag: --home
helm.go:76: [debug] unknown flag: --home
Error: looks like "git+https://bitbucket.org/sambatv/coresvcs-charts@?ref=master" is not a valid chart repository or cannot be reached: plugin "helm-git" exited with error
```

I have no idea what is triggering that error.  As can be seen by my example, I'm not specifying `--home` flag anywhere.

Has anyone had any success with Helm 3.0.0 and this helm-git plugin?

Many thanks in advance.

**Comment by bacongobbler** [comment]
can you provide a link to the plugin?

**Comment by mojochao** [comment]
It's the helm-git plugin repo https://github.com/aslafy-z/helm-git.

**Comment by bacongobbler** [comment]
Yeah, it's from the plugin. It's relying on a feature that was removed in helm 3.

https://github.com/aslafy-z/helm-git/blob/master/helm-git-plugin.sh#L81

**Comment by mojochao** [comment]
This has been fixed in helm-git v0.5.0.  Closing issue.  Thanks for the diagnostics!
